src/aktualizr_secondary/socket_server.cc

```diff
@@ -7,7 +7,7 @@
 #include "utilities/dequeue_buffer.h"
 #include "utilities/sockaddr_io.h"
 
-#include <linux/tcp.h>
+#include <netinet/tcp.h>
```

Comment:
Does this work on Linux as well? That would be cool.

Reply:
It looks like it.  The CI failures were because of a clang-format problem, so hopefully this will pass.